ocp_install_env.sh

```diff
@@ -276,6 +276,15 @@ sshKey: |
 fips: ${FIPS_MODE:-false}
 EOF
 
+  if [[ ! -z "$INSTALLER_PROXY" && "$IP_STACK" = "v4" ]]; then
```

Comment:
Should we output an error if INSTALLER_PROXY and ipv6 ? 
otherwise its silently ignored